SoftDevSix/ArgosUI
# fix: ensure production image is built with VITE ENV vars

fixes #38
* * * *

## What's in this pull request?




The image is being built without the VITE ENV vars which result in a
defective image that cannot communicate with other services.

- Add a validation to ensure ENV vars are present.
- Update prod compose file with ENV build arguments.


## Type of change



## How Has This Been Tested?



1. Build and run a new image `docker compose -f docker-compose-prod.yml up -d --build`
2. Run argos-api and argos-filelamanger services
3. Load a new project using the UI

Make sure to use production environmental variables:
```
VITE_FILE_MANAGER_API_BASE_URL=http://prod-argos.westus2.cloudapp.azure.com:8080
VITE_ARGOS_API_BASE_URL=http://prod-argos.westus2.cloudapp.azure.com:8081
```

## Checklist:
